ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -296,13 +296,6 @@ rm /tmp/pull-secret
 
 # custom rhcos ami for non-public regions
 RHCOS_AMI=
-if [ "$REGION" == "us-gov-west-1" ] || [ "$REGION" == "us-gov-east-1" ] || [ "$REGION" == "cn-north-1" ] || [ "$REGION" == "cn-northwest-1" ]; then
-  # TODO: move repo to a more appropriate location
-  curl -sL https://raw.githubusercontent.com/yunjiang29/ocp-test-data/main/coreos-for-non-public-regions/images.json -o /tmp/ami.json
```

Comment:
@yxun the _embedded_ AMIs are being used by GovCloud, so please update https://github.com/openshift/release/pull/36150/files#diff-fa12c2c52dcf6920a7e6d88d84100fb3e67c3ba36df80070e042047ed51bf925R273 as well. `ocp-test-data` is no longer supported.

Reply:
Thanks for letting us know. I have updated this change in the PR below.